scripts/rki-update.py

```diff
@@ -19,38 +19,35 @@
     print("You can install them with:  pip install pandas dfply")
     sys.exit(1)
 
-if len(sys.argv) != 2:
+if len(sys.argv) != 3:
     print("Usage:  python rki-update.py [RKI-filename.csv]")
```

Comment:
You changed the number of args but didn't change the help, which is now incorrect

Reply:
Fixed on commit [e411baf](https://github.com/matsim-vsp/covid-sim/pull/51/commits/e411baff386ac145b8b3aae92b0b1d2044dd8268)